image.yaml

```diff
@@ -97,10 +98,11 @@ ports:
     - value: 8443
 modules:
       repositories:
+          - path: modules
           - name: cct_module
             git:
-                  url: https://github.com/jboss-openshift/cct_module.git
-                  ref: master
+                  url: https://github.com/luck3y/cct_module.git
+                  ref: eap-online
```

Comment:
This shouldn't be changed.  For development work, you can use an overrides.yaml file to override the location of the cct_module repository (e.g. `path: /home/me/src/cct_module`.

Reply:
Yep, this is just for my  dev work. I don't think we actually have any cct_module changes (I did at one point).